wiki/Beatmap_Editor/Beat_Snap_Divisor/en.md

```diff
@@ -1,123 +1,113 @@
 
-The 1/3 and 1/6 are special because it means your BPM is divisible by three, but this is *rare*.
+Due to its versatility, this snap divisor is extremely popular among mappers and is frequently used in all sorts of maps in general. It is also the default setting that is being designated by the editor upon importing a new audio file.
 
-Holding down the `Alt` button over the slider changes it to the [distance spacing](/wiki/Beatmap_Editor/Distance_Snap) slider
+## 1/5 snap divisor
 
-## osu!mania
+![1/5 snap divisor in the osu!/osu!taiko/osu!catch editor](img/1_5_snap_divisor.png "1/5 snap divisor in the osu!/osu!taiko/osu!catch editor")
 
-Double white line: Measure
+![1/5 snap divisor in the osu!mania editor](img/1_5_snap_divisor_m.png "1/5 snap divisor in the osu!mania editor")
 
-White line: Common Time
+The 1/5 snap divisor allows for hit objects to be placed on the fifth fraction of a beat and its multiples (e.g. 1/5, 2/5, 3/5, and 4/5). It is being represented by yellow-coloured ticks on the editor timeline.
 
-Green line: You.
+This snap divisor is extremely uncommon out there and should be used with extreme care; in case you find yourself resorting to 1/5 snap divisor out of the blue, it is recommended for you to seek further advice from experienced mappers.
 
-### Divisor 1/1
+## 1/6 snap divisor
 
-![Divisor 1/1](/wiki/shared/1_1_m.jpg "Divisor 1/1")
+![1/6 snap divisor in the osu!/osu!taiko/osu!catch editor](/wiki/shared/BSD_1_6.jpg "1/6 snap divisor in the osu!/osu!taiko/osu!catch editor")
 
-- Full Beats
-- White ticks only
-- Best used for Easy difficulty
+![1/6 snap divisor in the osu!mania editor](/wiki/shared/1_6_m.jpg "1/6 snap divisor in the osu!mania editor")
 
-### Divisor 1/2
+The 1/6 snap divisor allows for hit objects to be placed on *double triplets*, which is the sixth fraction of a beat and its multiples (e.g. 1/6 and 5/6). It is being represented by purple-coloured ticks on the editor timeline.
 
-![Divisor 1/2](/wiki/shared/1_2_m.jpg "Divisor 1/2")
+Although relatively rare, there are some songs and music genres out there (e.g. swing jazz) that are primarily utilizing this snap divisor throughout.
 
-- Half Beats
-- Adds red ticks
+## 1/7 snap divisor
 
-### Divisor 1/3
+![1/7 snap divisor in the osu!/osu!taiko/osu!catch editor](img/1_7_snap_divisor.png "1/7 snap divisor in the osu!/osu!taiko/osu!catch editor")
 
-![Divisor 1/3](/wiki/shared/1_3_m.jpg "Divisor 1/3")
+![1/7 snap divisor in the osu!mania editor](/wiki/shared/1_7_m.jpg "1/7 snap divisor in the osu!mania editor")
 
-- Triplets
-- Has two purple ticks between each white tick
-- Compound Timing
+The 1/7 snap divisor allows for hit objects to be placed on the seventh fraction of a beat and its multiples (e.g. 1/7, 2/7, 3/7, and so on). It is being represented by yellow-coloured ticks on the editor timeline.
```

Comment:
```suggestion
The 1/7 snap divisor allows for hit objects to be placed on the seventh fraction of a beat and its multiples (e.g. 1/7, 2/7, 3/7, etc.). It is being represented by yellow-coloured ticks on the editor timeline.
```

Reply:
don't really fancy having two abbreviations (`e.g.` and `etc.`) this close to each other so i'll be keeping this as well i guess

(this doesn't change the point of the sentence anyway lol)